src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,51 @@
+import { useSearchParams, Link, useNavigate } from 'react-router-dom';
+import classNames from 'classnames';
+import { getSearchWith } from '../utils/searchHelper';
+
 export const PeopleFilters = () => {
+  const [searchParams] = useSearchParams();
+  const centuries = searchParams.getAll('centuries');
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex');
+  const navigate = useNavigate();
+
+  const onQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    navigate({
+      search: getSearchWith(searchParams, {
+        query: event.target.value || null,
+      }),
+    });
+  };
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
-        <a className="is-active" href="#/people">
+        <Link
+          className={`${sex ? '' : 'is-active'}`}
```

Comment:
Use the classnames library rules for add classes by conditions, fix it everywhere